src/js/components/FileInput/doc.js

```diff
@@ -18,6 +18,13 @@ export const doc = (FileInput) => {
       .description('Whether the control is disabled.')
       .defaultValue(undefined),
     id: PropTypes.string.description('The id attribute of the input.'),
+    maxSize: PropTypes.oneOfType([
+      PropTypes.number,
+      PropTypes.shape({
+        unit: PropTypes.string,
+        size: PropTypes.number,
+      }),
+    ]).description('Specifies a file size limit for the FileInput'),
```

Comment:
Doc should be updated on grommet-site repo, see https://github.com/grommet/grommet-site/pull/248